README.md

```diff
@@ -119,7 +119,7 @@ Deploy PGO to watch Postgres clusters in all of your [namespaces][k8s-namespaces
 [disaster-recovery]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/disaster-recovery/
 [high-availability]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/high-availability/
 [monitoring]: https://access.crunchydata.com/documentation/postgres-operator/v5/tutorial/monitoring/
-[multiple-cluster]: https://access.crunchydata.com/documentation/postgres-operator/v5/architecture/high-availability/multi-cluster-kubernetes/
+[multiple-cluster]: https://access.crunchydata.com/documentation/postgres-operator/v5/architecture/disaster-recovery/#standby-cluster-overview
```

Comment:
This page existed in v4, but the closest match in v5 seems to be this other page

Reply:
Agree this is the proper v5 equivalent :+1: